CHANGELOG.md

```diff
@@ -2,6 +2,53 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 42 end 2016-06-20
+   
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+42+Ending+June+20%2C+2016%22+label%3Aenhancement)
+
+- Providers
+  - Containers: Limit number of concurrent SmartState Analyes
+  - OpenStack
+      - Cleanup of Nova services after scale down
+      - Prevent retired instances from starting
+  - Hawkular
+      - Added missing fields in UI to improve user experience
+      - Middleware as top level menu item
+      - Default view for Middleware is datasource
+- Platform: Appliance Console
+  - Limited menu when running inside a container
+  - Removed menu items that are not applicable when running inside a container
+- Automate
+  - Engine: Allow arguments in method calls during substitution 
+  - Policy: Built-in policy to prevent retired VM from starting on a resume power operation
+  - Service Model: Expose provision_priority value
+  - Retirement: Restored retirement logic to verify that VM was provisioned or contains Lifecycle tag before processing
+  - Added lifecycle tag as a default tag
+  - Ansible
```

Comment:
1. Ansible is a provider, so it belongs in the Provders section.  (side note, @gmcculloug can we please change this is Sprint demo for next time?)
2. The provider should be Ansible Tower, which is technically what we integrate with.